qcut/docs/issues/kling-01/error3.md

```
@@ -0,0 +1,97 @@
+use-ai-generation.ts:874 
+📦 Starting generation for 1 models
+use-ai-generation.ts:948 step 4: sanitized params for kling_o1_ref2video Object
+use-ai-generation.ts:953 
+🎬 [1/1] Processing model: kling_o1_ref2video (Kling O1 Reference-to-Video)
+use-ai-generation.ts:976 step 5: sending generation request for kling_o1_ref2video (avatar tab) Object
+use-ai-generation.ts:1590   🎭 Calling generateAvatarVideo for kling_o1_ref2video with reference image...
+ai-video-client.ts:2803 🎭 Starting avatar video generation with FAL AI
+ai-video-client.ts:2804 🎬 Model: kling_o1_ref2video
+ai.tsx:567 [AI View] Progress: 0% - Generating with Kling O1 Reference-to-Video (1/1)
+ai-video-client.ts:2892 📤 Uploading reference image to FAL...
+index.html#/editor/82080feb-400d-4fb7-bc15-873dbbc28a7b:1 Access to fetch at 'https://fal.run/upload' from origin 'app://.' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)